sample/F8/tracks

```diff
@@ -0,0 +1,209 @@
+[{
+		"id": 1,
+		"name": "ADS & MONETIZATION",
```

Comment:
I just tried doing that. Git doesn't identify any change and so I can't even push it. @iamareebjamal